app/src/main/java/space/taran/arknavigator/mvp/model/repo/scores/PlainScoreStorage.kt

```diff
@@ -53,23 +53,14 @@ class PlainScoreStorage(
         if (!scoreById.containsKey(id))
             error("Storage isn't aware of this resource id")
 
-        if (score <= 0) {
```

Comment:
We allow negative scores indeed, but we should not store 0 scores since that could make the storage excessively big. `0` should be default value.

Reply:
This is handled, no zero score is persisted